src/__tests__/Swipeable.spec.js

```diff
@@ -379,4 +380,125 @@ describe('Swipeable', () => {
       expect(swipeableDiv.prop('onTouchMove')).toBe(instance.eventMove);
     });
   });
+
+  describe('Handle Rotation by 90 degree: ', () => {
```

Comment:
Thank you for adding tests!!!

Reply:
'course - won't work without. ;)

Reply:
ah, a question: the tests are mainly data-driven. Is it okay to refactor them into data-driven tests with less duplication?

Reply:
Yeah, externally